demisto_sdk/commands/test_content/xsiam_tools/xsiam_client.py

```diff
@@ -126,6 +127,18 @@ def _session(self) -> requests.Session:
     def _session(self, value: requests.Session):
         self.__session = value
 
+    def get_demisto_version(self) -> str:
```

Comment:
```suggestion
    @retry(times=5, exceptions=(....)
    def get_demisto_version(self) -> str:
```
better to cache the results here instead of calling the api over and over again for no reason if we do that ATM.
See retry decorator examples.